src/features/common/local-testing-benefits.mdx

```diff
@@ -0,0 +1,8 @@
+Testing locally using mock contracts saves you time and resources during development. Some of the key benefits include:
```

Comment:
What's the reason for single-sourcing this as common? It's fine to leave it, but I only see it used in one place.

Reply:
@dwightjl it's used in 2 diffrent files:

- src/pages/vrf/v2/subscription/examples/test-locally.mdx
- src/pages/vrf/v2/direct-funding/examples/test-locally.mdx